### DISCUSSION TITLE
Settare a Pagato uno solo dei pagamenti di un Received Document

### DISCUSSION THREAD

**Discussion by dcesca** [fattureincloud/api#427] [Q&A] [answered]
Se io ho un received document con più di un pagamento (magari la classica fattura a 30/60/90) e io devo mettere a pagato uno solo di questi, devo passare tutto l’elenco dei pagamenti con solo quello da modificare cambiato? O posso mandare solo quello cambiato e FiC fa il match con la due_date e importo?
È possibile avere un esempio concreto?
Sto usando l’sdk PHP

Grazie del supporto

**Comment by emajo** [comment, marked as answer]
Ciao @dcesca, per modificare un pagamento risulta necessario ripassare tutti i pagamenti del documento con le modifiche apportate a quello interessato, per fare ciò è necessario fare prima una chiamata [Get Received Document](https://github.com/fattureincloud/fattureincloud-php-sdk/blob/master/docs/Api/ReceivedDocumentsApi.md#getreceiveddocument), modificare il payment item come necessario ed effettuare una chiamata [Modify Received Document](https://github.com/fattureincloud/fattureincloud-php-sdk/blob/master/docs/Api/ReceivedDocumentsApi.md#modifyreceiveddocument).

Grazie,
Emanuele.
